apps/api/src/app/rate-limiting/usecases/evaluate-api-rate-limit/evaluate-api-rate-limit.types.ts

```diff
@@ -35,4 +35,8 @@ export type EvaluateApiRateLimitResponseDto = {
    * The cost of the request.
    */
   cost: number;
+  /**
+   * The API service level used to evaluate the request.
+   */
+  apiServiceLevel: string;
```

Comment:
Intentionally type-widening here, as the evaluation doesn't need to know about the specific enum typing of the api service level, it is simply proxying the value.